src/app/(proper_react)/(redesign)/(public)/page.tsx

```diff
@@ -14,9 +14,10 @@ import {
 import { getEnabledFeatureFlags } from "../../../../db/tables/featureFlags";
 import { getL10n } from "../../../functions/server/l10n";
 import { View } from "./LandingView";
+import { authOptions } from "../../../api/utils/auth";
 
 export default async function Page() {
-  const session = await getServerSession();
+  const session = await getServerSession(authOptions);
   if (typeof session?.user.email === "string") {
```

Comment:
@rhelmer do you also want to change this to `fxa_uid`?

Reply:
Yes I think we should for consistency, now that we know it will work...